### PR TITLE
build(licence.md): created LICENCE.MD with licence MIT

### PR DESCRIPTION
**Pull request by Ricefab02**

### CODE REVIEW

File: LICENCE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) <year> <copyright holders>
```
Comment (rocambille) [inline LICENCE.md]: you need to change here `<year>` and `<copyright holders>` ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: LICENCE.md (hunk 1 above)
Comment (Ricefab02) [reply LICENCE.md]: ok it's good

File: LICENCE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) <2020> <Brian THELLIER, Stephaine PANTIEZ, Fabrice ROGER  holders>
```
Comment (rocambille) [inline LICENCE.md]: holders should be removed to ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: LICENCE.md (hunk 2 above)
Comment (Ricefab02) [reply LICENCE.md]: ok !!  :)

Comment (rocambille) [reply LICENCE.md]: still waiting for the fix :)

**Review by rocambille — CHANGES_REQUESTED** [review]

**Pull request merged**